assets/app/views/create/nextSteps.html

```diff
@@ -0,0 +1,102 @@
+<div ng-controller="ProjectController">
+  <div ng-controller="CreateFinishedController" class="container next-steps">
+    <ol class="breadcrumb">
+      <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
+      <li><a href="project/{{projectName}}/create">Add to Project</a></li>
+      <li ng-if="name"><a href="#">{{name}}</a></li>
+      <li class="active"><strong ng-if="pendingTasks(tasks()).length">Creating</strong><strong ng-if="!pendingTasks(tasks()).length">Completed</strong></li>
+    </ol>
+
+    <div ng-controller="TasksController">
+      <div class="page-header page-header-bleed-right">
+        <h1 ng-if="!tasks().length">Completed</h1>
+        <h1 ng-if="tasks().length && allTasksSuccessful(tasks())">Completed successfully</h1>
+        <h1 ng-if="pendingTasks(tasks()).length">Creating...</h1>
+        <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Completed, with errors</h1>
+      </div>
+
+      <div class="buttons" ng-if="!tasks().length">
+        <a class="btn btn-primary btn-lg" href="project/{{projectName}}/overview">Go to overview</a>
+      </div>
+
+      <div ng-repeat="task in tasks()">
+        <h3>
+          <i class="spinner spinner-xs spinner-inline" ng-show="task.status=='started'"></i>
+          {{ task | taskTitle }}.
+        </h3>
+
+        <div ng-repeat="alert in task.alerts">
```

Comment:
do we think we'll re-use this "small" format for alerts anywhere else?  if so we should probably move this into the alerts directive and have it take an optional format parameter, defaulting to the existing larger format